docs/source/tutorials/datalab.ipynb

```diff
@@ -0,0 +1,1393 @@
+      "       outlier 0.522080           6\n",
+      "\n",
+      "(Note: A lower score indicates a more severe issue across all examples in the dataset.)\n",
+      "\n",
+      "\n",
+      "----------------------- label issues -----------------------\n",
+      "\n",
+      "Number of examples with this issue: 11\n",
+      "Overall dataset quality in terms of this issue: : 0.9091\n",
+      "\n",
+      "Examples representing most severe instances of this issue:\n",
+      "     is_label_issue  label_score given_label predicted_label\n",
+      "77             True     0.006939        high             mid\n",
+      "7              True     0.007830         low             mid\n",
+      "40             True     0.014826         mid             low\n",
+      "107            True     0.021220        high             mid\n",
+      "120            True     0.026403        high             mid\n",
+      "\n",
+      "\n",
+      "------------------ near_duplicate issues -------------------\n",
+      "\n",
+      "Number of examples with this issue: 8\n",
+      "Overall dataset quality in terms of this issue: : 0.2465\n",
+      "\n",
+      "Examples representing most severe instances of this issue:\n",
+      "     is_near_duplicate_issue  near_duplicate_score near_duplicate_sets  distance_to_nearest_neighbor\n",
+      "131                     True          0.000000e+00               [123]                  0.000000e+00\n",
+      "123                     True          0.000000e+00               [131]                  0.000000e+00\n",
+      "129                     True          4.463180e-07               [130]                  4.463180e-07\n",
+      "130                     True          4.463180e-07               [129]                  4.463180e-07\n",
+      "51                      True          3.857172e-02                [52]                  3.859087e-02\n",
+      "\n",
+      "Additional Information: \n",
+      "threshold: 0.04392298874881288\n",
+      "\n",
+      "\n",
+      "---------------------- outlier issues ----------------------\n",
+      "\n",
+      "Number of examples with this issue: 6\n",
+      "Overall dataset quality in terms of this issue: : 0.5221\n",
+      "\n",
+      "Examples representing most severe instances of this issue:\n",
+      "     is_outlier_issue  outlier_score  nearest_neighbor  distance_to_nearest_neighbor\n",
+      "126              True       0.046465                 5                  2.353282e+00\n",
+      "130              True       0.068695               129                  4.463180e-07\n",
+      "129              True       0.068695               130                  4.463180e-07\n",
+      "127              True       0.076251                76                  2.297110e+00\n",
+      "128              True       0.083941                57                  2.199371e+00\n"
+     ]
+    }
+   ],
+   "source": [
+    "lab.report()"
```

Comment:
@elisno I thought this would still print the 1-2 sentence summary of what each issue type means?
I.e. the descriptions I helped you truncate before.

Let's revert to that behavior by default, with an optional flag user can specify = True to suppress these explanations (but the flag should be false by default)

Reply:
I'm sorry, the flag was actually flipped the other way around a few days ago. I'll revert it.

On that note: The flag is `include_description: bool = True` by default. The explanations can be suppressed by setting it to `False`.